dockerfile

```diff
@@ -0,0 +1,47 @@
+# Use python:buster image as base
+FROM python:buster as base
```

Comment:
It's safer to specify a version of Python. They aren't always backwards compatible and currently your docker image will suddenly switch to Python 3.10 one day